apps/geo_to_hca.py

```diff
@@ -104,20 +95,39 @@ def srr_fastq(srr_accessions):
         return xml,xml_content,url
 
     @staticmethod
-    def srr_experiment(srr_accession):
-        sleep(0.5)
-        srr_experiment_url = rq.get(f'https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch/fcgi?db=sra&id={srr_accession}')
-        if srr_experiment_url.status_code == 400:
-            raise NotFoundSRA(srr_experiment_url, srr_accession)
-        return xm.fromstring(srr_experiment_url.content)
+    def split_list(accessions, n):
+        parts_list = []
+        for i in range(0, len(accessions), n):
+            part = accessions[i:i + n]
+            if part is not None:
+                parts_list.append(part)
+        return parts_list
 
     @staticmethod
-    def srr_biosample(biosample_accession):
-        sleep(0.5)
-        srr_biosample_url = rq.get(f'https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch/fcgi?db=biosample&id={biosample_accession}')
-        if srr_biosample_url.status_code == 400:
-            raise NotFoundSRA(srr_biosample_url, biosample_accession)
-        return xm.fromstring(srr_biosample_url.content)
+    def request_info(accessions,accession_type):
+        if accession_type == 'biosample':
+            url = f'https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch/fcgi?db=biosample&id={",".join(accessions)}'
+        if accession_type == 'experiment':
+            url = f'https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch/fcgi?db=sra&id={",".join(accessions)}'
+        sra_url = rq.get(url)
+        if sra_url.status_code == 400:
+            raise NotFoundSRA(sra_url, accessions)
+        return xm.fromstring(sra_url.content)
+
+    @staticmethod
+    def get_content(accessions,accession_type):
+        if len(accessions) < 100:
```

Comment:
there's a fair number of places where there's comparison against either a string literal or a number. Do you think it would be a good idea to make constants out of these values for better code readability?